integrations/tensorflow/test/iree_tfl_tests/llvmcpu_mobilenet_v1.run

```diff
@@ -1,2 +1,3 @@
 # REQUIRES: llvmcpu
 # RUN: %PYTHON -m iree_tfl_tests.mobilenet_v1_test --target_backend=llvmcpu --artifacts_dir=%t
+# XFAIL: *
```

Comment:
While XFAIL is generally useful, I would actually exclude the test here with `REQUIRES`, using this pattern: https://github.com/iree-org/iree/blob/e0cd5b83e7e15053a4db977d60ed16f262d34e2c/integrations/tensorflow/test/iree_tfl_tests/cartoon_gan.run#L1-L3

The specific file this test is trying to download is https://github.com/iree-org/iree/blob/e0cd5b83e7e15053a4db977d60ed16f262d34e2c/integrations/tensorflow/test/python/iree_tfl_tests/mobilenet_v1_test.py#L11

That URL is returning an error. We have a tracking issue covering migrating `iree-model-artifacts` GCP files to other locations: https://github.com/iree-org/iree/issues/18518 . 

I'd rather keep the test disabled until we take intentional action to move the file, instead of XFAIL it and then have the test maybe start unexpectedly passing if the URL becomes live again.